ci-operator/config/openshift/knative-client/openshift-knative-client-release-v1.2.0.yaml

```diff
@@ -95,7 +95,7 @@ tests:
     owner: openshift-ci
     product: ocp
     timeout: 4h0m0s
-    version: "4.10"
+    version: "4.9"
```

Comment:
so... I have the feeling that w/ 4.10 things were _worse_ than w/ 4.9

Maybe just me... but I have that feeling....

Reply:
I can't tell either from client jobs perspective. No significant difference noticed except the recent issue with pool capacity being exhausted.